### DISCUSSION TITLE
Unexpected exception trying to send to websocket

### DISCUSSION THREAD

**Discussion by stavrt** [Chia-Network/chia-blockchain#5136] [Support]
In the debug.log file I keep getting these errors:

2021-05-15T09:45:15.819 daemon __main__                   : ERROR    Unexpected exception trying to send to websocket: code = 1005 (no status code [internal]), no reason Traceback (most recent call last):
  File "chia\daemon\server.py", line 187, in safe_handle
  File "websockets\protocol.py", line 555, in send
  File "websockets\protocol.py", line 803, in ensure_open
websockets.exceptions.ConnectionClosedError: code = 1005 (no status code [internal]), no reason

I also noticed that my average response time to the harvesters is very high now. Currently I have a Windows PC as a full node and 3 harvesters connected using the CA cert method described in the docs page.

**Comment by AkiraDX** [comment]
I'm also having exactly the same issue.

My node keep dropping out of sync.

**Comment by chiamaster** [comment]
any updates? Shouldn't this be listed in issue/bug and not discussion?

**Comment by Atompeng** [comment]
Here are some bug reports regarding this -
https://github.com/Chia-Network/chia-blockchain/issues/2810
https://github.com/Chia-Network/chia-blockchain/issues/5100

This has been reported as a bug twice and confirmed by a number of people, but both reports have been closed for some reason. I believe @noisymime found a solution in their comment on issue 2810; I'll be testing it over the weekend.

**Reply by GeorgePapageorgakis** [reply]
Is this fixed? I see it too.

**Reply by vazmuten** [reply]
Still see it ALL THE TIME under MS Windows ...

**Reply by loppefaaret** [reply]
https://github.com/Chia-Network/chia-blockchain/issues/2810#issuecomment-851198398
> Sargonas wrote: It appears to be a case of your operating system and python collectively not cleaning up after themselves, and leaving certain sockets in use after the programs terminate processes uncleanly. For the time being the only practical solution is to simply first try to restart your chia daemon and if that fails to address it then reboot your system.

**Comment by BrandtH22** [comment]
@stavrt , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
